src/object/srv_obj.c

```diff
@@ -1770,6 +1830,21 @@ ds_obj_tgt_update_handler(crt_rpc_t *rpc)
 
 	rc = dtx_end(&dth, ioc.ioc_coc, rc);
 	obj_rw_reply(rpc, rc, ioc.ioc_map_ver, 0, ioc.ioc_coc);
+	if (ioc.ioc_free_sgls) {
```

Comment:
The buffer is allocated for inline fetch, why it's freed in ds_obj_tgt_update_handler()? Shouldn't we do this free in ds_obj_rw_handler()?

Reply:
ah, crap. Thanks.